src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/SnapshotResolver.cs

```diff
@@ -17,15 +18,15 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.ProjectSystem;
 internal class SnapshotResolver : ISnapshotResolver
 {
     private readonly ProjectSnapshotManagerAccessor _projectSnapshotManagerAccessor;
-    private readonly ILogger<SnapshotResolver> _logger;
+    private readonly ILogger _logger;
 
     // Internal for testing
     internal readonly HostProject MiscellaneousHostProject;
 
-    public SnapshotResolver(ProjectSnapshotManagerAccessor projectSnapshotManagerAccessor, ILoggerFactory loggerFactory)
+    public SnapshotResolver(ProjectSnapshotManagerAccessor projectSnapshotManagerAccessor, IRazorLoggerFactory loggerFactory)
```

Comment:
```
IRazorLoggerFactory loggerFactory)
```

if it is an `IRazorLoggerFactory` then the naming should reflect that or else it gets confusing. (same comment for other places where naming does not match the type).

That would have made the PR diff much larger but if it makes sense please consider renaming some of them later.

Reply:
I think I disagree with this one. It is still a logger factory, the fact that we have `Razor` in the name is just a disambiguation. If we didn't reference MS.Ext.Logging, I would have called it `ILoggerFactory`.